pkg/crd/testdata/cronjob_types.go

```diff
@@ -499,6 +499,7 @@ type CronJobStatus struct {
 // +kubebuilder:subresource:status
 // +kubebuilder:resource:singular=mycronjob
 // +kubebuilder:storageversion
+// +kubebuilder:additionalmetadata:annotations="api-approved.kubernetes.io=https://github.com/kubernetes-sigs/controller-tools";"cert-manager.io/inject-ca-from-secret=cert-manager/cert-manager-webhook-ca"
```

Comment:
```suggestion
// +kubebuilder:metadata:annotations="api-approved.kubernetes.io=https://github.com/kubernetes-sigs/controller-tools";"cert-manager.io/inject-ca-from-secret=cert-manager/cert-manager-webhook-ca"
```

Do we need the additional? 
Why not only annotations? 
So, if set we add annotations informed 
If not set, we add only the default annotation

Reply:
Emm.. okey, I will update it to `// +kubebuilder:metadata:annotations=...`

Reply:
Updated